miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1456,55 +1444,21 @@ public static QcType to(QcTypeDto from) {
     return to;
   }
 
-  public static SampleQcDto asDto(SampleQC from) {
-    SampleQcDto dto = new SampleQcDto();
+  public static QcDto asDto(QC from) {
+    QcDto dto = new QcDto();
     dto.setId(from.getId());
-    dto.setQcDate(getDateString(from.getQcDate()));
-    dto.setQcCreator(from.getQcCreator());
-    dto.setQcType(asDto(from.getQcType()));
+    dto.setDate(getDateString(from.getDate()));
+    dto.setCreator(from.getCreator().getFullName());
+    dto.setType(asDto(from.getType()));
     dto.setResults(from.getResults());
-    dto.setSampleId(from.getSample().getId());
+    dto.setOwnerId(from.getEntity().getId());
```

Comment:
Other places in MISO use `owner` to refer to a user, but it's used here to refer to the associated QC-able thing. What do you think about changing the field name on the DTO from `owner` to `entity` to match the QC model?